crates/bevy_tasks/README.md

```diff
@@ -11,6 +11,20 @@ or ordering of spawned tasks.
 It is based on [`async-executor`][async-executor], a lightweight executor that allows the end user to manage their own threads.
 `async-executor` is based on async-task, a core piece of async-std.
 
+## Usage
+
+Tasks are spawned via thread pools, of which bevy_tasks provides three different ones for different kinds of tasks.
```

Comment:
Ok, cool, I am prepared to learn more and to improve the docs even more. However, instead of just taking your suggestion as it is, I'd like to ask back what happens in a single-threaded (WASM) environment? I thought a "Usage" section should probably give specific instructions to beginners. As a beginner, I would now wonder "if I have tasks, how should I spawn them in a WASM build?"

Reply:
WASM environments right now are inherently single-threaded. You still use the same APIs, they just won't run on separate threads.

Reply:
I have now elaborated on single- vs. multi-threaded environments and WASM as well. (I have heard that threading support for WASM is planned, so I formulated the text in a way that it should still not read totally wrong in five years.)